config/passport.js

```diff
@@ -0,0 +1,39 @@
+const bcrypt = require('bcryptjs');
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const passportJWT = require('passport-jwt');
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJWT = passportJWT.ExtractJwt;
+const SECRET_KEY = '321cba';
+
+const db = require("./db");
+
+passport.use(new LocalStrategy(
+    (username, password, done) => {
+    db.query('SELECT * FROM users WHERE email = ?', username, (err, rows) => {
+        if (err) {
+            return done(err)
+        }
+        const userAccount = rows[0];
+        const passwordMatch  = bcrypt.compare(password, userAccount.password);
```

Comment:
Need to check if the db query returned nothing. Server crashes when an attempting to access 'userAccount.password'

Reply:
fixed